spec/predicates/null.md

```diff
@@ -0,0 +1,29 @@
+# Predicate type: Null
+
+Type URI: https://in-toto.io/Null
```

Comment:
I assume the idea here is that this Type URI does not need to be versioned because it will only ever be a field-less object?

Reply:
Yup, that's the thinking. Should I explain this, or is it sufficient as-is?

Reply:
I think it's probably worth being explicit given that the parsing rules state
> "Each type has a SemVer2 version number and the TypeURI reflects the major version number."

Reply:
Sent #33. Think I need something here, too?

Reply:
As discussed in #33, I added the version number here.